packages/components/src/navigation/index.js

```diff
@@ -28,6 +29,8 @@ export default function Navigation( {
 	const [ item, setItem ] = useState( activeItem );
 	const [ menu, setMenu ] = useState( activeMenu );
 	const [ slideOrigin, setSlideOrigin ] = useState();
+	const [ navigationItems, setNavigationItems ] = useState( null );
+	const [ navigationMenus, setNavigationMenus ] = useState( null );
```

Comment:
I know I've set the wrong example with the variable names of the original PR, but I think that adding yet another variation of `*item` and `*menu` is making everything very confusing. 😅 

This is not an issue for this specific PR, though. Just something to keep in mind, and maybe try to come up with clearer names.

Reply:
Since items and menus are not primitives anyway, what about "wrapping" them into a `navigationTree` object?
I think it would convey their intended use more clearly.

E.g.
```js
const navigationTree = { items, menus };
```

Reply:
Definitely agreed on the `navigationTree` naming and organization. Btw. do we even need a separation between `items` and `menus` in it?